com.unity.render-pipelines.universal/Shaders/LitForwardPass.hlsl

```diff
@@ -135,8 +135,14 @@ half4 LitPassFragment(Varyings input) : SV_Target
     InputData inputData;
     InitializeInputData(input, surfaceData.normalTS, inputData);
 
-    half4 color = UniversalFragmentPBR(inputData, surfaceData.albedo, surfaceData.metallic, surfaceData.specular, surfaceData.smoothness, surfaceData.occlusion, surfaceData.emission, surfaceData.alpha);
-    
+    half4 color = UniversalFragmentPBR(inputData, surfaceData.albedo, surfaceData.metallic, surfaceData.specular, surfaceData.smoothness, surfaceData.occlusion, surfaceData.emission, surfaceData.alpha
```

Comment:
What do you think if we add an override of this function that takes a SurfaceData as input. This way, if we need to add more surface data properties we don't need to change the function or add keywords.

Adding keywords to function signature is not a good approach anyway imo. It leads to confusion.

Reply:
I can add that.

Reply:
Added this. Resulted to additional dependency with SurfaceInput.hlsl. Will push soon.